db/migrate/schema.rb

```diff
@@ -1,58 +0,0 @@
-# This file is auto-generated from the current state of the database. Instead
-# of editing this file, please use the migrations feature of Active Record to
-# incrementally modify your database, and then regenerate this schema definition.
-#
-# Note that this schema.rb definition is the authoritative source for your
-# database schema. If you need to create the application database on another
-# system, you should be using db:schema:load, not running all the migrations
-# from scratch. The latter is a flawed and unsustainable approach (the more migrations
-# you'll amass, the slower it'll run and the greater likelihood for issues).
-#
-# It's strongly recommended that you check this file into your version control system.
-
-ActiveRecord::Schema.define(version: 2021_02_03_005802) do
-
-  # These are extensions that must be enabled in order to support this database
-  enable_extension "plpgsql"
-
-  create_table "dealers", force: :cascade do |t|
-    t.string "name"
-    t.boolean "fully_staffed"
-    t.integer "monthly_gross"
-    t.string "city"
-    t.datetime "created_at"
-    t.datetime "updated_at"
-    t.string "state"
-  end
-
-  create_table "instruments", force: :cascade do |t|
-    t.string "name"
-    t.boolean "on_sale"
-    t.string "brand"
-    t.integer "dealer_id"
-    t.integer "price"
-    t.datetime "created_at"
-    t.datetime "updated_at"
-  end
-
-  create_table "schools", force: :cascade do |t|
-    t.string "name"
-    t.string "address"
-    t.string "city"
-    t.string "state"
-    t.integer "zipcode"
-    t.boolean "gradeschool"
-    t.datetime "created_at"
-    t.datetime "updated_at"
-  end
-
-  create_table "teachers", force: :cascade do |t|
-    t.string "name"
-    t.integer "school_id"
-    t.boolean "college_graduate"
-    t.integer "salary"
-    t.datetime "created_at"
-    t.datetime "updated_at"
-  end
-
-end
```

Comment:
Just curious, what happened to this file? lol

Reply:
Sorry I didn't mention in the original comment but there was a copy for some reason! This was inside the migrations folder, I deleted it since there should only be one schema next to the seeds file

Reply:
Oh okay sweet!